package.json

```diff
@@ -4,6 +4,9 @@
   "author": "Valora Inc",
   "license": "Apache-2.0",
   "private": true,
+  "engines": {
+    "node": "^20"
+  },
```

Comment:
This enforces the use of node 20.x (not <20, but not >20).

Reply:
Should we enforce this in `.npmrc` file via `engine-strict=true`? https://docs.npmjs.com/cli/v10/using-npm/config#engine-strict

Reply:
`yarn` works with only engines. Is there an advantage to using `.npmrc`?

Reply:
Pardon for the mistake. I think I was wrong as `.npmrc` do not seem to be used in `yarn` projects.

I've seen projects also use Volta instead of nvm, and they define the engine version and Volta's node version separately in the package.json (https://runme.dev/blog/nodejs-version-managers-nvm-volta-asdf). It automatically sets the correct node version and environment without using `nvm use`. It may be nice to look into in the future.